src/protocol/packet.rs

```diff
@@ -84,11 +78,6 @@ impl<'a> PacketBuf<'a> {
     /// the header/checksum trimming stage. ASCII validation is still performed.
```

Comment:
stale comment:

```suggestion
    /// the header/checksum trimming stage.
```

There's also a similar stale comment referring to ASCII validation on line 43, in the doc comment for `new`. Can you remove that as well?